storage/include/s3/client.hpp

```diff
@@ -204,13 +204,13 @@ class Client : public concord::storage::IDBClient {
  protected:
   // retry forever, increasing the waiting timeout until it reaches the defined maximum
   template <typename F, typename... Args>
-  void do_with_retry(Status& r, F&& f, Args&&... args) const {
+  void do_with_retry(const std::string& msg, Status& r, F&& f, Args&&... args) const {
```

Comment:
Nit: Maybe we can consider changing the parameter to `std::string_view msg` to avoid copying and potential allocation from string literals.

Reply:
Sorry I missed this comment. Will open a new PR.

Reply:
https://github.com/vmware/concord-bft/pull/978